sentry-ruby/lib/sentry/net/http.rb

```diff
@@ -77,7 +77,10 @@ def from_sentry_sdk?
       end
 
       def extract_request_info(req)
-        uri = req.uri || URI.parse("#{use_ssl? ? 'https' : 'http'}://#{address}#{req.path}")
+        # IPv6 url could look like '::1/path', and that won't parse without
+        # wrapping it in square brackets.
+        hostname = address =~ Resolv::IPv6::Regex ? "[#{address}]" : address
```

Comment:
I think you need `require "resolv"` to use it.

Reply:
Yup, checked in `irb`, you actually have to require things before using them. Who knew. 

Seems like on some Rubies, it gets required within `net/http`, but it's not guaranteed. Let's see if explicit require solves it!